src/index.js

```diff
@@ -1,10 +1,20 @@
-import React from "react";
-import ReactDOM from "react-dom";
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Предлагаю сделать так, как дока рекомендует
```suggestion
import { createRoot } from 'react-dom/client';
```

А то у нас ошибка в консоли